automation/taskcluster/lib/tasks.py

```diff
@@ -16,6 +16,13 @@
 _OFFICIAL_REPO_URL = 'https://github.com/mozilla-mobile/fenix'
 _DEFAULT_TASK_URL = 'https://queue.taskcluster.net/v1/task'
 GOOGLE_APPLICATION_CREDENTIALS = '.firebase_token.json'
+# Bug 1558456 - Stop tracking youtube-playback-test on motoG5 for >1080p cases
+TEST_URL_PARAMS = {
+    "urlparams": [
```

Comment:
This data structure seems more advanced than it needs to be - what do you think about:
```
ARM_RAPTOR_URL_PARAMS = [
    "exclude=1,2,...,96",
]
```
I'm guessing that this logic was moved from another project with more refined URL param requirements

Reply:
Yes, i've used the logic from https://hg.mozilla.org/mozilla-central/rev/7c40e15ffdac#l2.18
ARM_RAPTOR_URL_PARAMS is fine too, i will make the changes and update the PR.
Thanks!